pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.609.1</version>
+    <version>2.25</version>
```

Comment:
Yes, I think this will bump it up because of how https://github.com/jenkinsci/plugin-pom brings in Jenkins versions.